adapter/noop2/noop2.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 Istio Authors.
```

Comment:
The adapter author doesn't see handlers at all. A handler is merely a bag of adapter config state and an adapter name.